calibr8/optimization.py

```diff
@@ -247,6 +247,8 @@ def fit_scipy_global(
     if not fit.success or bound_hit:
         _log.warning(f"Fit of {type(model).__name__} has failed:")
         _log.warning(fit)
+    else:
+        _log.warning(fit)
```

Comment:
Why do you always want to show a warning?

Reply:
Working with `fit_scipy()` and `fit_scipy_global()`, I realized that fitting attempts that raise the `hit_bound` warning were more informative. The warning provides critical information (termination message, fun, solver status, etc.) that the user should use to determine whether the fit was successful or not. 
Since `fit_scipy()` and `fit_scipy_global()` only return fit.x and history, we are currently missing that critical solver output. I added the warning to discuss a proper implementation of extended solver output.

Reply:
As long as there is no abnormal termination, the solver will consider the run successful.  
👇 This (exaggerated) example shows how a detailed termination statement can help identifying a unsuccessful fit.
![grafik](https://user-images.githubusercontent.com/97220522/191295882-5030ed19-3a9e-40c6-9837-a1c631cfe2c5.png)

We can address this issue in a separate pull request however.

Reply:
👍 yes, a separate PR would be good

Extending the condition with `not x.success or np.isnan(x.fun)` might do the trick